index.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 var numberIsNan = require('number-is-nan');
 
-function createArg(key, val) {
+function createArg(key, val, shortargs) {
+	if (typeof shortargs === 'object' && shortargs[key]) {
+		return '-' + shortargs[key] + (val ? ' ' + val : '')
+	}
```

Comment:
It would be better to handle the conversion to shortform on line 28. Then this could only be responsible for handling one character options and we wouldn't have to pass the `shortargs` argument.